cloud/deployment/deployment.go

```diff
@@ -393,7 +397,11 @@ func Create(name, workspaceID, description, clusterID, runtimeVersion, dagDeploy
 			case strings.ToLower(string(astrocore.CreateStandardDeploymentRequestSchedulerSizeEXTRALARGE)):
 				standardDeploymentRequest.SchedulerSize = astroplatformcore.CreateStandardDeploymentRequestSchedulerSizeEXTRALARGE
 			case "":
-				standardDeploymentRequest.SchedulerSize = astroplatformcore.CreateStandardDeploymentRequestSchedulerSize(configOption.DefaultValues.SchedulerSize)
+				if standardDeploymentRequest.RemoteExecution != nil && standardDeploymentRequest.RemoteExecution.Enabled {
```

Comment:
Standard deployment should have remote execution as nil right? or am I missing something?

Reply:
Yes, theoretically, but the API spec doesn't tell us that. I discussed that with @ianbuss, and we think having less business logic (which it is if it is not part of the OpenAPI spec) is better.